preview/action.yml

```diff
@@ -6,25 +6,31 @@ inputs:
     required: true
   localstack-api-key:
     description: 'LocalStack API key used to create the preview environment'
-    required: true
+    required: false
```

Comment:
any particular reason as to why this was changed?

Reply:
To give the option to provide the API key as an env variable as it's done with the other actions. If I force it as required I must feed in the env variable every time I call other pro related actions rather just inherit it from the top level.